### ISSUE TITLE
xcodeproj version requirement conflicts with CocoaPods >= 1.0.1

### ISSUE THREAD

**Issue by nikolaykasyanov** [SlatherOrg/slather#235]
```
Bundler could not find compatible versions for gem "xcodeproj":
  In snapshot (Gemfile.lock):
    xcodeproj (= 1.0.0)

  In Gemfile:
    cocoapods (= 1.0.1) was resolved to 1.0.1, which depends on
      xcodeproj (< 2.0, >= 1.1.0)

    fastlane (~> 1.0) was resolved to 1.97.2, which depends on
      xcodeproj (< 2.0.0, >= 0.20)

    slather (~> 2.0.2) was resolved to 2.0.2, which depends on
      xcodeproj (< 1.1.0, >= 0.28.2)
```

**Comment by nikolaykasyanov** [comment]
@neonichu 👍

**Comment by neonichu** [comment]
Shipped as 2.3.0 🚢
